packages/toc-extension/src/index.ts

```diff
@@ -66,7 +66,7 @@ async function activateTOC(
   app: JupyterFrontEnd,
   docmanager: IDocumentManager,
   editorTracker: IEditorTracker,
-  labShell: ILabShell,
+  labShell: ILabShell | null,
```

Comment:
The order of the dependencies is relevant, so we need to move that line down below `translator`.